src/tests/entities/customer-info.test.ts

```diff
@@ -441,4 +441,121 @@ describe("customer info parsing", () => {
     const customerInfo = toCustomerInfo(subscriberResponse);
     expect(customerInfo).toEqual(expectedCustomerInfo);
   });
+
+  test("subscriber info with paddle subscription purchase is parsed correctly", () => {
```

Comment:
TBH, not sure if we need a test for every store type... but I guess it doesn't hurt